docs/introduction.mdx

```diff
@@ -63,10 +63,10 @@ The ["@emotion/core"](https://www.npmjs.com/package/@emotion/core) package requi
 
 - CSS prop support
 
-  - Similar to the `style` prop but adds support for nested selectors, media queries, and auto-prefixing.
```

Comment:
I understand how lack of this information in the previous section could be confusing - but was this one here confusing as well? I would keep this section as is - mention of `style` prop is kinda nice, it indicates how it looks like in the code.

Reply:
Upon re-reading this part of the code, I realize that I was associating the `style` prop with the framework agnostic section for some reason. Let me re-add this part.